PatternPal/PatternPal.Extension/ViewModels/StepByStepListViewModel.cs

```diff
@@ -29,17 +30,41 @@ public class StepByStepListViewModel : ViewModel
             NavigateHomeCommand = new NavigateCommand< HomeViewModel >(
                 navigationStore,
                 () => new HomeViewModel(navigationStore));
+            
+            // Next button 
             NavigateStepByStepInstructionsCommand =
                 new NavigateCommand< StepByStepInstructionsViewModel >(
                     navigationStore,
                     () => new StepByStepInstructionsViewModel(
                         navigationStore,
-                        SelectedInstructionSet));
+                        SelectedInstructionSet,
+                        false));
+            
+            // Continue button
+            NavigateContinueStepByStepInstructionsCommand =
+                new NavigateCommand<StepByStepInstructionsViewModel>(
+                    navigationStore,
+                    () => new StepByStepInstructionsViewModel(
+                        navigationStore,
+                        SelectedInstructionSet,
+                        true));
+
+            GetInstructionSetsResponse response = 
+                GrpcHelper.StepByStepClient.GetInstructionSets(new GetInstructionSetsRequest());
+
+            List<Recognizer> temFilteredRecognizers = response.Recognizers.Where(r => r is Recognizer.Singleton).ToList();
+            // Dropdown menu in the SBS viewmodel
+            InstructionSetList = temFilteredRecognizers;
+
+            Recognizer test = InstructionSetList.FirstOrDefault();
 
-            GetInstructionSetsResponse response = GrpcHelper.StepByStepClient.GetInstructionSets(new GetInstructionSetsRequest());
+            GetInstructionSetResponse response2 =
```

Comment:
Don't use generic variable names like `response`, `response2`, ect. Instead use more speaking variable names like `instructionSetsResponse` and `instructionSetResponse`.

Reply:
Variable names of these have changed among other variable names such as temp, test, and such.